### DISCUSSION TITLE
Custom Lightmaps?

### DISCUSSION THREAD

**Discussion by ghost** [theomenden/TheOmenDen.Vanadium#46] [Q&A] [answered]
Hi! Do you intend on having Vandium support custom Lightmaps, similar to Optifine and Colormatic? Definently one of the more overlooked things in custom resourcepacks but they can make all the difference in aesthetic/tone without using shaders.

Thanks!

**Comment by AluTheCrow** [comment, marked as answer]
It does have the code for custom lightmaps, and does support them - however other issues - such as the way we interface with sodium are not really allowing us to demonstrate that. 

This however, is in progress to be fixed asap.
